src/scancode/cli.py

```diff
@@ -278,6 +282,7 @@ def scancode(ctx,
     scanners = OrderedDict(zip(possible_scans.keys(), zip(possible_scans.values(), scan_functions)))
 
     scans_cache_class = get_scans_cache_class()
+    user_ignore = dict((inputs, 'Ignore files while scanning, that are supplied by --ignore') for inputs in ignore)
```

Comment:
Use a dict comprehension instead. Do not repeat the help text of the command option